intellij/org.eclipse.xtext.idea/src/org/eclipse/xtext/idea/documentation/IdeaDeclarationDocumentationProvider.xtend

```diff
@@ -37,10 +37,10 @@ class IdeaDeclarationDocumentationProvider {
 
 	def protected getFileInfo(PsiEObject element) {
 		val resource = element.EObject.eResource
-		val module = workspaceCfgProvider.getWorkspaceConfig(resource.resourceSet).findProjectContaining(resource.URI)
-		if (module !== null) {
-			val uri = resource.URI.deresolve(module.path) 
-			return '''[�module.name�] �uri�'''
+		val projectConfig = projectConfigProvider.getProjectConfig(resource.resourceSet)
```

Comment:
Here we used to show the project name of the referenced thing. Now it's the project of the resource set, e.g. the file that is currently shown in the editor.